### ISSUE TITLE
bug: Focus is not getting trapped for alert dialogue using talkback in ionic 5 

### ISSUE THREAD

**Issue by mastergogo** [ionic-team/ionic#21598]
We using ionic 5 and we noticed that on the alert dialogue while navigating using Talkback, talkback focus does not get trapped on the alert, and instead it goes behind the dialogue if the user swipes forward. this seems to be fixed in ionic 3 but not ionic 5. here is the PR #11543  
Will look forward to the response.

**Comment by liamdebeasi** [comment]
Thanks for the issue. Please update to the latest version of Ionic Framework. This was fixed in Ionic Framework v5.1.0.

**Comment by mastergogo** [comment]
Thanks for the reply, upgrading ionic version didn't work for us, the focus is still not getting trapped on alert dialogue, we have created the project using ionic and Vue, upgrading to the new version should also work for us?

**Comment by liamdebeasi** [comment]
We are still working on Ionic Vue, so this fix should be available in the next beta. We were waiting for the Vue 3 API to become stable before continuing to work on it. Now that it is stable, we are continuing development on Ionic Vue.

**Comment by mastergogo** [comment]
Thanks again for informing, could you please help with the timelines of the next beta as in when it is out? just so I can inform my team and look forward to the fix?

**Comment by mastergogo** [comment]
Hi Liam, I also checked the [alert](https://ionic-docs-demo.herokuapp.com/component/alert) which is part of the core (ionic 5) I believe, and focus is not getting trapped here as well for the talk back, attached is the screenshot for reference.
![alert-talkback-error](https://user-images.githubusercontent.com/8082610/85544528-b89de200-b638-11ea-9a71-ee95c997b96d.jpeg)

**Comment by mastergogo** [comment]
Hi @liamdebeasi, have u had the chance to check the above comment? we need to know if these issues can get fixed at the framework level as writing code explicitly can cause issues later when we upgrade to newer versions.

**Comment by liamdebeasi** [comment]
The issue was fixed in a recent version of Ionic Framework, but the docs demo you linked to is using an outdated version.

As for Ionic Vue, we are working on it right now, and hope to have a release candidate in the next few months. We do not recommend using the old betas of Ionic Vue in production.

**Comment by mastergogo** [comment]
Hi @liamdebeasi, thank you for the clarification, could you please share the latest docs demo version? so we can test and implement them accordingly. Thanks again!

**Comment by liamdebeasi** [comment]
The latest version is Ionic Framework v5.2.2.

You can install using `npm install @ionic/angular@latest` if using Angular of `npm install @ionic/react@latest @ionic/react-router@latest` if using React.

**Comment by mastergogo** [comment]
Hi @liamdebeasi, We upgraded our core to the latest (V5.2.2) as you mentioned and tested with talk back but the focus is still not getting trapped in the alert dialogue and goes behind the modal dialogue if u swipe forward, I think this is because there is no `aria-hidden` given to the parent window once the modal dialogue is opened,  I also tested few apps that are featured on the ionic website like Sworkit, JustWatch and untapped and noticed that focus goes behind the modal, as it will be a blocker for us could you please help?

**Comment by liamdebeasi** [comment]
Can you please provide a repo I can use to reproduce the issue?

**Comment by mastergogo** [comment]
Hi @liamdebeasi, I have attached a zip, let me know if u can run talkback (Android screen reader) after extracting the files from the zip, as I am new to all ionic and JS frameworks, I created this small project by learning from a Udemy course and have also updated the ionic core to latest 5.2.3, below are the steps to reproduce:

1) Launch the app
2) turn on the talkback (from settings>accessibility>talkback )
3) navigate to any of the two recipes using talkback (by swiping right on the screen)
4) double-tap to go inside any of the recipes 
5)  Navigate to the delete icon and double-tap to activate the alert.
6) once the alert is triggered, navigate to the delete button by swiping right
7) when the talkback focus is on the delete button, keep swiping right, after two or three swipes talkback focus will go behind the alert.

I couldn't reproduce this in the browser with the latest Ionic core, checked with NVDA as talkback is only for Android, however, it is reproducible in devices. Let me know if u are able to reproduce and thanks in advance.

[Ionic+talkback-issue.zip](https://github.com/ionic-team/ionic-framework/files/4926846/Ionic%2Btalkback-issue.zip)
